core/domain/suggestion_services_test.py

```diff
@@ -939,6 +945,90 @@ def setUp(self):
+                'cmd': 'add_story_node',
+                'node_id': 'node_1',
+                'title': 'Node1',
+            }), story_domain.StoryChange({
+                'cmd': 'update_story_node_property',
+                'property_name': 'exploration_id',
+                'node_id': 'node_1',
+                'old_value': None,
+                'new_value': self.EXP_ID
+            })], 'Added exploration.')
+
+    def create_translation_suggestion_associated_with_exp(
+            self, exp_id, author_id):
+        """Creates a translation suggestion that is associated with an
+        exploration with id exp_id. The author of the created suggestion is
+        author_id.
+        """
+        # Gets the html content in the exploration to be translated.
+        exploration = exp_fetchers.get_exploration_by_id(exp_id)
+        content_html = exploration.states['State 1'].content.html
+
+        add_translation_change_dict = {
+            'cmd': exp_domain.CMD_ADD_TRANSLATION,
+            'state_name': 'State 1',
+            'content_id': 'content',
+            'language_code': 'hi',
+            'content_html': content_html,
+            'translation_html': '<p>This is translated html.</p>'
+        }
+
+        suggestion_services.create_suggestion(
+            suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            suggestion_models.TARGET_TYPE_EXPLORATION,
+            exp_id, 1, author_id, add_translation_change_dict,
+            'test description')
+
+        # Check that there is only one suggestion associated with the
+        # exploration and that the suggestion is out for review.
+        suggestions = suggestion_services.query_suggestions(
+            [('author_id', self.author_id), ('target_id', self.EXP_ID)])
+        self.assertEqual(len(suggestions), 1)
+        self.assertEqual(
+            suggestions[0].status, suggestion_models.STATUS_IN_REVIEW)
+
+    def test_get_translation_suggestions_with_exp_ids(self):
+        # Create translation suggestion associated with the exploration
+        # that was created during set up.
+        self.create_translation_suggestion_associated_with_exp(
+            self.EXP_ID, self.author_id)
+
+        # Create another exploration.
+        self.save_new_linear_exp_with_state_names_and_interactions(
+            'exp2', self.editor_id, ['State 1', 'State 2'],
+            ['TextInput'], category='Algebra')
+
+        self.create_translation_suggestion_associated_with_exp(
+            'exp2', self.author_id)
+        self.assertEqual(
+            len(suggestion_services.get_translation_suggestions_with_exp_ids(
+                [self.EXP_ID])), 1)
+        self.assertEqual(
+            len(suggestion_services.get_translation_suggestions_with_exp_ids(
+                ['bad_exp_id'])), 0)
+        self.assertEqual(
+            len(suggestion_services.get_translation_suggestions_with_exp_ids(
+                ['exp2', self.EXP_ID])), 2)
```

Comment:
Try to organize tests into sections best you can, i.e. ARRANGE (test setup), ACT (action/method under test), ASSERT (assertions to be made). Usually this means separating sections by whitespace. This makes tests easier to scan and read. The wiki has a bit of info on this as well: [link](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests). Here and below.

Reply:
I moved this test to the SuggestionGetServicesUnitTests class because I thought it made more sense there. I used mocks instead of real explorations and I spilt the test up into several tests so that it was more clear. I changed the spacing too. Let me know what you think!